src/components/Home.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable react/react-in-jsx-scope */
```

Comment:
Try to avoid disabling lintern rules.

If you do not agree with some of the rules, I suggest removing them instead.